src/candela/components/ScatterPlot/index.js

```diff
@@ -85,6 +85,19 @@ export default class ScatterPlot extends VisComponent {
   }
 
   render () {
-    vega.parseChart(spec, this.el, this.options);
+    this.chart = vega.parseChart(spec, this.el, this.options);
+  }
+
+  getSerializationFormats () {
+    return ['png', 'svg'];
+  }
+
+  serialize (format) {
+    if (!this.chart) {
+      this.render();
+    }
+    return this.chart.then((vobj) => {
```

Comment:
I'd say get rid of the parentheses around `vobj` (unless we decided on a style that keeps them for single-argument arrow functions).